src/main/resources/errors.yaml

```diff
@@ -300,6 +300,16 @@ request-errors:
       
       Resend the command using a filter.
 
+  - scope: FILTER
+    code: UNSUPPORTED_FILTER_FOR_UPDATE_ONE_DELETE_ONE
+    title: updateOne and deleteOne commands do not support filter operation other than `$eq`
+    body: |-
+      The Filtering using the non `$eq` filter operations can select more than one row, and so cannot be used with the `updateOne` and `deleteOne` commands as they can only modify one row.
+      
+      The command used an invalid filter on the columns: ${unsupportedFilterColumns}.
+      
+      Resend the command only using `$eq` filter.
```

Comment:
maybe: "filter" -> "filters" ? (since one can have more than one)
also maybe rephrase "command only using" -> "command that only has"

Reply:
Resend the command using only `$eq` filters.